docs/src/content/components/errors.mdx

```diff
@@ -21,16 +16,16 @@ import {
 
 ## Examples
 
-### 403 Forbidden
+### 403 Forbidden (size=small)
 
 [403 Forbidden](https://httpstatuses.com/403) The requested resource could not be found but may be available in the future. Subsequent requests by the client are permissible.
 
 ```typescript
-import ForbiddenErrorPage from '@pluralsight/ps-design-system-errors'
+import {ForbiddenErrorPage} from '@pluralsight/ps-design-system-errors'
```

Comment:
Looks like this will resolve https://github.com/pluralsight/design-system/issues/1488 👍